### ISSUE TITLE
Contribution guidelines

### ISSUE THREAD

**Issue by spences10** [chungu-voyage-bears-team-16/huddlehunt#3]
Directions on branching and how to GitHub in a team environment.

This should be a way for contributors to understand the GitHub workflow:

* Create a branch
* Add commits
* Open a PR
* Discuss and review code with a teammate
* Deploy

I'll add documentation around the associated git commands used for this, and also for VSCode, as that's my text editor of choice :ok_hand: :sunglasses: 

Check the [GitHub workflow](https://guides.github.com/introduction/flow/) link as well

**Comment by thomasmarr** [comment]
Thanks. I assume PR is a Push Request? I use VS Express and/or notepad++, but I imagine VS Express will be similar to VS Code anyway

**Comment by spences10** [comment]
A Pull Request apologies I should have made that clear :upside_down_face: PR is when you can merge the fork/feature branch into the master branch.

With regard to VS Express I haven't used it for JS or and Dev other than VB.NET and C# so I couldn't say, I have been asked a few times now to do a getting started with VSCode post so maybe this could go into that.

VSCode is like notepad++ on steroids I'd recommend checking it out :ok_hand:

**Comment by thomasmarr** [comment]
Yeah, a friend of mine was showing me VSCode recently and it seemed to be a
trimmed down version of VS Express (now known as VS Community - I think).
